website/pages/api-docs/events.mdx

```diff
@@ -362,265 +365,158 @@ http://127.0.0.1:4646/v1/event/stream
-                "driver.docker": "true",
-                "driver.docker.bridge_ip": "172.17.0.1",
-                "driver.docker.os_type": "linux",
-                "driver.docker.runtimes": "runc",
-                "driver.docker.version": "19.03.13"
-              },
-              "Detected": true,
-              "Healthy": true,
-              "HealthDescription": "Healthy",
-              "UpdateTime": "2020-10-15T10:07:37.904159516-04:00"
-            },
-            "exec": {
-              "Attributes": null,
-              "Detected": false,
-              "Healthy": false,
-              "HealthDescription": "Driver must run as root",
-              "UpdateTime": "2020-10-15T10:07:37.445083368-04:00"
-            },
-            "java": {
-              "Attributes": null,
-              "Detected": false,
-              "Healthy": false,
-              "HealthDescription": "Driver must run as root",
-              "UpdateTime": "2020-10-15T10:07:37.445601605-04:00"
-            },
-            "mock_driver": {
-              "Attributes": {
-                "driver.mock": "true"
-              },
-              "Detected": true,
-              "Healthy": true,
-              "HealthDescription": "Healthy",
-              "UpdateTime": "2020-10-15T10:07:37.445193068-04:00"
-            },
-            "qemu": {
-              "Attributes": null,
-              "Detected": false,
-              "Healthy": false,
-              "HealthDescription": "",
-              "UpdateTime": "2020-10-15T10:07:37.445684857-04:00"
-            },
-            "raw_exec": {
-              "Attributes": {
-                "driver.raw_exec": "true"
-              },
-              "Detected": true,
-              "Healthy": true,
-              "HealthDescription": "Healthy",
-              "UpdateTime": "2020-10-15T10:07:37.445431163-04:00"
-            }
-          },
-          "CSIControllerPlugins": null,
-          "CSINodePlugins": null,
-          "HostVolumes": null,
-          "CreateIndex": 7,
-          "ModifyIndex": 8
-        }
-      }
-    }
-  ]
+  "ID": "webhook",
+  "Address": "http://127.0.0.1:8080",
+  "Type": "webhook",
+  "Topics": {
+    "*": ["*"]
+  }
 }
+
+$ curl \
+    -X POST http://localhost:4646/v1/event/sink/webhook -d @event-sink.json
+```
+
+## Get an Event Sink
+
+This endpoint returns the current configuration details of a specific event sink.
+
+| Method | Path                       | Produces           |
+| ------ | -------------------------- | ------------------ |
+| `GET`  | `/v1/event/sink/:sink_id`  | `application/json` |
+
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs#blocking-queries) and
+[required ACLs](/api-docs#acls).
+
+| Blocking Queries | ACL Required         |
+| ---------------- | -------------------- |
+| `YES`            | `operator:read`      |
+
+### Sample Request
+
+```shell-session
+$ curl \
+    http://localhost:4646/v1/event/sink/deployment-webhook
```

Comment:
Where does the `deployment-webhook` sink come from? It might be better to use only examples which relate to the sink that's being created in the create example.

Reply:
I can add another create example so it is more apparent why its there